gremlin-go/run.sh

```diff
@@ -41,10 +41,14 @@ fi
 
 # Parses current gremlin server version from project pom.xml file using perl regex
 GREMLIN_SERVER_VERSION=$(grep tinkerpop -A2 pom.xml | grep -Po '(?<=<version>)([0-9]+\.?){3}(-SNAPSHOT)?(?=<)')
-GREMLIN_SERVER="${1:-$GREMLIN_SERVER_VERSION}"
+export GREMLIN_SERVER="${1:-$GREMLIN_SERVER_VERSION}"
 echo "$GREMLIN_SERVER"
 
+ABS_PROJECT_HOME=$(dirname $(realpath "$0"))/..
+export ABS_PROJECT_HOME
+echo "$ABS_PROJECT_HOME"
```

Comment:
debug info?

Reply:
More or less. Should I remove it?

Reply:
Remove or make more meaningful like `Absolute project home is ....`

Reply:
Ok!

Reply:
removed this one, and made the above one more meaningful